packages/file-uploader/src/components/fileuploader/Preview.tsx

```diff
@@ -0,0 +1,337 @@
+  border-top: 1px solid ${({ theme }) => theme.colors.neutral06};
+  padding: 16px;
+  font-size: ${({ theme }) => theme.typeScale.size02.fontSize};
+  line-height: ${({ theme }) => theme.typeScale.size02.lineHeight};
+`;
+
+const StyledFlex = styled(Flex)`
+  height: 56px;
+  padding: 0 8px 0 16px;
+  font-size: ${({ theme }) => theme.typeScale.size02.fontSize};
+  line-height: ${({ theme }) => theme.typeScale.size02.lineHeight};
+`;
+
+const FileName = styled(Flex)`
+  overflow: hidden;
+  white-space: nowrap;
+  align-items: center;
+  text-overflow: ellipsis;
+  display: block;
+  margin-right: 24px;
+  font-size: ${({ theme }) => theme.typeScale.size02.fontSize};
+  line-height: ${({ theme }) => theme.typeScale.size02.lineHeight};
+`;
+
+const StyledCard = styled(Card)<{ file: FilePreview; isInverse: boolean }>`
+  background-color: ${({ isInverse, theme }) =>
+    isInverse ? theme.colors.foundation02 : theme.colors.neutral08};
+  border-color: ${({ file, theme, isInverse }) =>
+    file.errors
+      ? isInverse
+        ? theme.colors.dangerInverse
+        : theme.colors.danger
+      : theme.colors.neutral06};
+  border-width: ${({ file }) => (file.errors ? '2px' : '1px')};
+  margin: 10px 0;
+`;
+
+const ErrorHeader = styled.span`
+  display: block;
+
+  > div {
+    display: flex;
+    align-self: center;
+    margin-right: 12px;
+  }
+`;
+
+const ErrorMessage = styled.span`
+  display: block;
+`;
+
+const formatError = (
+  error: { header?: string; message: string; code: string },
+  constraints: {
+    maxSize?: number;
+    minSize?: number;
+    accept?: string | string[];
+  }
+) => {
+  switch (error.code) {
+    case 'file-too-large':
+      return {
+        ...error,
+        message: `${error.message} ${formatFileSize(constraints.maxSize)}.`,
+      };
+    case 'file-too-small':
+      return {
+        ...error,
+        message: `${error.message} ${formatFileSize(constraints.minSize)}.`,
+      };
+    case 'file-invalid-type':
+      const accept =
+        Array.isArray(constraints.accept) && constraints.accept.length === 1
+          ? constraints.accept[0]
+          : constraints.accept;
+      const messageSuffix = Array.isArray(accept)
+        ? `one of ${accept.join(', ')}`
+        : accept;
+      return { ...error, message: `${error.message}: ${messageSuffix}` };
+    default:
+      return error;
+  }
+};
+
+export const Preview = forwardRef<HTMLDivElement, PreviewProps>(
+  (props, ref) => {
+    const {
+      accept,
+      file,
+      isInverse: isInverseProp,
+      maxSize,
+      minSize,
+      onDeleteFile,
+      onRemoveFile,
+      testId,
+      thumbnails,
+      ...rest
+    } = props;
+
+    const theme: ThemeInterface = useContext(ThemeContext);
+    const i18n: I18nInterface = React.useContext(I18nContext);
+    const isInverse = useIsInverse(isInverseProp);
+    const [actions, setActions] = useState(<CloseIcon />);
+
+    const handleRemoveFile = (file: FilePreview) => {
```

Comment:
small thing, but in all of our other components we use the `function {x}` format